src/Request.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+
+class Request {
+  constructor (event) {
+    this.request = this.createRequest(event)
+  }
+
+  getRequest () {
+    return this.request
+  }
+
+  createRequest (event) {
```

Comment:
I suggest a refactoring that will made the code more readable

```
const MYSTERIOUS_CONSTANT_IN_FILTER_VALUE_IN_TEMPLATES = '%';

class Request {
  constructor (event) {
    this.request = this.createRequest(event)
  }

  getRequest () {
    return this.request
  }

  createRequest (event) {
    const params = event.queryStringParameters
    const template = this.isJournal(event) 
        ? this.readJournalTemplate() 
        : this.readPublisherTemplate()
    
    let filterValue = this.crateQueryFilterBasedOnQueryParameters(params);
    this.doSomeExtraMagic(template, filterValue);
    return template
  }

  doSomeExtraMagic(template, filterValue) {
    template.filter[0].selection.values[0] = filterValue
  }

  crateQueryFilterBasedOnQueryParameters(params) {
    let filterValue = MYSTERIOUS_CONSTANT_IN_FILTER_VALUE_IN_TEMPLATES
    if (this.queryHasNoParameters(params)) {
      filterValue = '%' + params.query + '%'
    }
    return filterValue;
  }

  queryHasNoParameters(params) {
    return params !== undefined && params.query !== undefined;
  }

  readPublisherTemplate() {
    return readTemplate('query_publisher_template.json');
  }

  readJournalTemplate() {
    return readTemplate('query_journal_template.json');
  }

  isJournal(event) {
    return event.path === '/journal';
  }
}
const readTemplate = (path) => JSON.parse(fs.readFileSync(path).toString())
```

Reply:
Renamed and refactored methods to clarify meaning